code/modules/item_worth/worths_list.dm

```diff
@@ -482,7 +483,6 @@ var/list/worths = list(
 					/obj/item/assembly = 50,
 					/obj/item/uv_light = 150,
 					/obj/item/eftpos = 30,
-					/obj/item = 90,
```

Comment:
Why remove `/obj/item` from here and below?

Reply:
I'm fairly sure the two lines of /obj/item used to be /obj/item/weapon/ and /obj/item/device.
Having it like this sets all items not otherwise specified to be worth 90 and that seems strange to me I guess?

Reply:
Oh that makes sense yeah.